Modix.Services/Promotions/CreateCampaignContinuationData.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using Discord;
+using Modix.Data.Models.Core;
+
+namespace Modix.Services.Promotions
+{
+    public class CreateCampaignContinuationData
+    {
+        public CreateCampaignContinuationData(
```

Comment:
The current paradigm for model classes in MODiX is to just make them pure POCO's. `{ get; set; }` properties and default constructors. IMO, initializing an object through `{ }` initializer syntax is WAY cleaner than through a constructor, and access control modifiers exist for the sake of business logic, not data.

Reply:
Works for me! Updated.